plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlConnectorTest.java

```diff
@@ -377,4 +377,15 @@ protected Session joinPushdownEnabled(Session session)
                 .setCatalogSessionProperty(session.getCatalog().orElseThrow(), "join_pushdown_strategy", "EAGER")
                 .build();
     }
+
+    @Test
+    @Override
+    public void testDateYearOfEraPredicate()
```

Comment:
Overwrite the test because : https://support.oracle.com/knowledge/Oracle%20Database%20Products/2725151_1.html
Also please reference : https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-16.html#mysqld-8-0-16-bug
```
When comparing [DATE](https://dev.mysql.com/doc/refman/8.0/en/datetime.html) values with constant strings, MySQL first tries to convert the string to a DATE and then to perform the comparison. When the conversion failed, MySQL executed the comparison treating the DATE as a string, which could lead to unpredictable behavior. Now in such cases, if the conversion of the string to a DATE fails, the comparison fails with [ER_WRONG_VALUE](https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html#error_er_wrong_value). (Bug #29025656)
```